src/css/main.less

```diff
@@ -733,6 +733,9 @@ fieldset[disabled] .btn-info.active {
     &.fixed-height {
       height: 30px;
     }
+    button, input {
```

Comment:
This shifts the buttons in all modals, also the dialogs, where the buttons are right aligned.
![buttons](https://cloud.githubusercontent.com/assets/859154/9636266/d315db9a-519a-11e5-855d-1a85dc9c8d2e.png)